src/services/SiteDashboardService.ts

```diff
@@ -1,11 +1,19 @@
 import {
   CollaboratorsStats,
+  LastUpdatedInfo,
   SiteDashboardInfo,
   SiteDashboardReviewRequest,
 } from "types/siteDashboard"
 
 import { apiService } from "./ApiService"
 
+export const getLastUpdated = async (
+  siteName: string
+): Promise<LastUpdatedInfo> => {
+  const endpoint = `/sites/${siteName}/lastUpdated`
```

Comment:
if possible endpoints should be made last-updated. but non-blocking, this shld be standardised for a couple of endpoints actually

Reply:
True that yeah, I think we can perform this migration after all the v1 endpoints have been upgraded to v2.